traits/tests/test_enum.py

```diff
@@ -13,10 +13,15 @@
 
 from traits.api import (
     Any, BaseEnum, Enum, HasTraits, List, Property, TraitError)
+from traits.etsconfig.api import ETSConfig
 from traits.testing.optional_dependencies import pyface, requires_traitsui
 
-if pyface is not None:
-    GuiTestAssistant = pyface.toolkit.toolkit_object(
+is_qt = ETSConfig.toolkit.startswith('qt')
```

Comment:
the copying of this here is related to this comment:
https://github.com/enthought/enable/pull/444#pullrequestreview-530204914

Reply:
Aha, the only GUI test here is skipped if `traitsui` is not available and can be rewritten using `UITester` like this:
```
# this opens and closes the GUI to exercise initialization of the default traitsui editor for Enum
with UITester().create_ui(obj):
    pass
```
So rewriting the test not to depend on GuiTestAssistant is also an alternative...

Reply:
I'm worried about compatibility constraints here, though. I'm not sure we want Traits 6.2.0 to depend on TraitsUI latest.

In practice, people upgrading to the latest set of ETS packages should be able to do it in two separate steps:

1. Upgrade Traits
2. Upgrade TraitsUI and Pyface.

OTOH, this is only in tests, so it's only the Traits _test_ suite that ends up requiring the latest TraitsUI. I guess that doesn't affect users.